File: TITULO PROPIO/objetos/utils/model/usuario.js
class Usuario {
  // propiedades
  nombre;
  apellido;
  edad;
  dni;

  // constructor

  constructor(nombreP, apellidoP, dniP, edadP) {
    this.nombre = nombreP;
    this.apellido = apellidoP;
    this.edad = edadP;
    this.dni = dniP;
  }

  // metodos

  mostrarDatos() {
    console.log(`Nombre ${this.nombre}`);
    console.log(`Apellido ${this.apellido}`);
    console.log(`DNI ${this.dni}`);
    console.log(`Edad ${this.edad}`);
  }

  getNombre() {
    return this.nombre;
  }

  setNombre(nombre) {
    this.nombre = nombre;
  }

  // getter setter
  set setApellido(apellido) {
    console.log("Llamando al meotodo set");
    this.apellido = apellido;
  }

  get getApellido() {
    console.log("Llamando al meotodo get");
    return this.apellido;
  }
}

File: T7/EscrituraObjetos/src/model/Usuario.java
package model;

import java.io.Serializable;

public class Usuario implements Serializable {

    private final static Long serialVersionUID = 1234L;

    private String nombre, apellido, dni;

    public Usuario() {
    }

    public Usuario(String nombre, String apellido, String dni) {
        this.nombre = nombre;
        this.apellido = apellido;
        this.dni = dni;
    }

    public void mostrarDatos(){
        System.out.println("nombre = " + nombre);
        System.out.println("apellido = " + apellido);
        System.out.println("dni = " + dni);
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getApellido() {
        return apellido;
    }

    public void setApellido(String apellido) {
        this.apellido = apellido;
    }

    public String getDni() {
        return dni;
    }

    public void setDni(String dni) {
        this.dni = dni;
    }
}
